site/en/docs/extensions/mv3/getstarted/tut-tabs-manager/index.md

```diff
@@ -0,0 +1,400 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./popup.css" />
+  </head>
+  <body>
+    <template id="li_template">
+      <li>
+        <a>
+          <h3 class="title">Tab Title</h3>
+          <p class="pathname">Tab Pathname</p>
+        </a>
+      </li>
+    </template>
+
+    <h1>Google Dev Docs</h1>
+    <button>Group Tabs</button>
+    <ul></ul>
+
+    <script src="./popup.js" type="module"></script>
+  </body>
+</html>
+```
+
+{% Aside %}
+
+💡 **TIP**: You can use [top level await][mdn-top-level] by adding `type="module”` to the script
+tag.
+
+{% endAside %}
+
+Next, let's style the popup. Create a popup CSS file and include the following code:
+
+{% Label %}popup.css:{% endLabel %}
+
+```css
+body {
+  width: 20rem;
+}
+
+ul {
+  list-style-type: none;
+  padding-inline-start: 0;
+  margin: 1rem 0;
+}
+
+li {
+  padding: 0.25rem;
+}
+li:nth-child(odd) {
+  background: #80808030;
+}
+li:nth-child(even) {
+  background: #ffffff;
+}
+
+h3,
+p {
+  margin: 0;
+}
+```
+
+### Step 3: Manage the tabs {: #step-3 }
+
+The [Tabs API][api-tabs] allows an extension to create, query, modify, and rearrange tabs in the
+browser.
+
+#### Request permission {: #narrow }
+
+You can use many methods in the Tabs API without requesting any permission. However, we need access to the `title` and the `URL` of the tabs; these sensitive properties require permission.
+
+We could request `"tabs"` permission, but this would allow access to the sensitive properties of **all** browser tabs. Since we are only managing tabs of a particular site, we will request narrow host permissions. 
+
+Narrow [host permissions][doc-match] allow us to protect user privacy by granting elevated permission to **specific sites**. This will grant access to the `title`, and `URL` properties, as well as additional capabilities.
+
+Add the following code to the `manifest.json` file:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  ...
+  "host_permissions": [
+    "https://developer.chrome.com/*"
+  ],
+  ...
+}
+```
+
+<!-- TODO: Details with each permission warning -->
+
+#### Query the tabs {: #query }
+
+You can retrieve the tabs from specific URLs using the `tabs.query()` method. Create a `popup.js`
+file and add the following code:
+
+{% Label %}popup.js:{% endLabel %}
+
+```js
+const tabs = await chrome.tabs.query({
+  url: [
+    "https://developer.chrome.com/docs/webstore/*",
+    "https://developer.chrome.com/docs/extensions/*",
+  ],
+});
+...
+```
+
+{% Details %}
+{% DetailsSummary %}
+💡 **Can I use Chrome APIs directly in the popup?**
+{% endDetailsSummary %}
+
+A popup and other extension pages can call any [Chrome API][doc-apis] because they are served from the
+chrome schema. For example `chrome-extension://EXTENSION_ID/popup.html`.
+
+{% endDetails %}
+
+#### Focus on a tab {: #focus}
+
+First, the extension will sort the list alphabetically. Then, when a list item is clicked, it will
```

Comment:
```suggestion
First, the extension will sort tab names (the titles of the contained HTML pages) alphabetically. Then, when a list item is clicked, it will
```

Reply:
Did I correctly infer what was being sorted? We need to say either way.